### ISSUE TITLE
steal's babel 5 is trying to load .babelrc (with custom transpile option)

### ISSUE THREAD

**Issue by whitecolor** [stealjs/steal-tools#363]
There is `.babelrc` with `Babel 6 options` in project folder. Custom `traspile` option is passed to steal-tools build. But it still tries to load `.babelrc.` and fails:

```
[BABEL]  Unknown option: .babelrc.presets 
```

**Comment by matthewp** [comment]
Not sure what you are talking about, can you provide some example code and more explanation. How do you know babel 5 is trying to load .babelrc, where are you getting this info from?

**Comment by whitecolor** [comment]
When build starts. Just place `.babelrc`
```
{
  "presets": [
    "es2015", "stage-0"
  ],
  "plugins": [
    "transform-runtime"
  ]
}
```
in project folder and start build.

**Comment by whitecolor** [comment]
And I think I should not touch it or if custom `transpile` is passed.

**Comment by matthewp** [comment]
Ok, I can't remember why we depend on babel but we shouldn't so I'll remove that.

**Comment by whitecolor** [comment]
Ok, thanks)

**Comment by whitecolor** [comment]
just remind as impossible to have `.babelrc` anywhere near with steal (it also scans parents).

**Comment by matthewp** [comment]
Should be fixed in 0.13.4
